emission/storage/timeseries/non_user_timeseries.py

```diff
@@ -5,10 +5,11 @@
 
 import emission.core.get_database as edb
 import emission.storage.timeseries.builtin_timeseries as bits
+import emission.core.wrapper.entry as ecwe  # Added missing import
 
 class NonUserTimeSeries(bits.BuiltinTimeSeries):
     def __init__(self):
-        super(AggregateTimeSeries, self).__init__(None)
+        super(NonUserTimeSeries, self).__init__(None)  # Corrected superclass name
```

Comment:
This is incorrect. We are defining the `NonUserTimeSeries` here, we can't also inherit from it at the same time.

Reply:
@shankari When I put `AggregateTimeSeries` instead of `NonUserTimeSeries`;

I get 
```
Traceback (most recent call last):
  File "/Users/rsimpson/e-mission-server/emission/tests/funcTests/TestFunctionTiming.py", line 95, in <module>
    main()
  File "/Users/rsimpson/e-mission-server/emission/tests/funcTests/TestFunctionTiming.py", line 92, in main
    execute_and_time_function(func)
  File "/Users/rsimpson/e-mission-server/emission/tests/funcTests/TestFunctionTiming.py", line 74, in execute_and_time_function
    store_dashboard_error(
  File "/Users/rsimpson/e-mission-server/emission/storage/decorations/stats_queries.py", line 78, in store_dashboard_error
    store_stats_entry(None, "stats/dashboard_error", code_fragment_name, ts, reading)
  File "/Users/rsimpson/e-mission-server/emission/storage/decorations/stats_queries.py", line 56, in store_stats_entry
    time_series = esta.TimeSeries.get_non_user_time_series()
  File "/Users/rsimpson/e-mission-server/emission/storage/timeseries/abstract_timeseries.py", line 30, in get_non_user_time_series
    return nuts.NonUserTimeSeries()
  File "/Users/rsimpson/e-mission-server/emission/storage/timeseries/non_user_timeseries.py", line 12, in __init__
    super(AggregateTimeSeries, self).__init__(None)
NameError: name 'AggregateTimeSeries' is not defined
```